analysis_for_run.py

```diff
@@ -0,0 +1,236 @@
+idds=list(tmp['Id'])
+
+# Download Reads from Run
+for i in idds:
+    bashCommand ="""
+    bs download dataset \
+            --config=%s \
+            --id=%s \
+            --output="/Volumes/IDGenomics_NAS/WGS_Serotyping/%s/Sequencing_reads/Raw" \
+            --retry
+    """ % (user,i,args.run_name)
+    process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+print("Reads downloaded into: /Volumes/IDGenomics_NAS/WGS_Serotyping/%s/Sequencing_reads/Raw" % args.run_name)
+slack_message("Reads downloaded into: /Volumes/IDGenomics_NAS/WGS_Serotyping/%s/Sequencing_reads/Raw" % args.run_name)
+
+if args.download_reads:
+    sys.exit()
+
+# Start ICA Anaylsis if not mycosnp; Call other python script
+if args.analysis == 'Grandeur':
+    subprocess.call("python auto_%s_ICA.py %s %s" % (args.analysis,args.run_name,args.ica_reference), shell=True)
+
+# While loop to look in ICA to See if run is finished
+slack_message('Started %s Anaylysis on ICA and Monitoring' % args.run_name)
+print('Started %s Anaylysis on ICA and Monitoring' % args.run_name)
+bashCommand = "icav2 projects enter Testing"
+process = subprocess.Popen(bashCommand.split(" ",3), stdout=subprocess.PIPE)
+process.communicate()
+t=0
+while t == 0:
+    bashCommand = "icav2 projectanalyses list --max-items 0"
+    tmp=icav_out(bashCommand)
+    tmp2=[]
+    for i in range(len(tmp)):
+        if args.run_name in tmp[i]:
+            tmp2.append(tmp[i])
+    try:
+        if tmp2[-1].split()[-1] == 'SUCCEEDED':
+            slack_message('%s is "SUCCEEDED" on ICA' % args.run_name)
+            print('%s is "SUCCEEDED" on ICA' % args.run_name)
+            t=1
+        if tmp2[-1].split()[-1] == 'FAILED':
+            slack_message('%s is "FAILED" on ICA' % args.run_name)
+            print('%s is "FAILED" on ICA' % args.run_name)
+            t=1
+            break
+    except:
+        continue
+    time.sleep(1200)
+
+slack_message("%s has comleted analysis on ICA" % args.run_name)
+print("%s has comleted analysis on ICA" % args.run_name)
+
+# Now that the analysis has 'SUCCEEDED', the most important files needed for analysis are downloaded
+if args.ica_download:
+    bashCommand = "icav2 projectdata list --file-name %s --match-mode FUZZY" % args.run_name
+    tmp= icav_out(bashCommand)
+    for i in tmp:
+       try:
+            bashCommand = "./icav2 projectdata list --parent-folder %sResults/" % i.split()[0]
+            tmp= icav_out(bashCommand)
+            target_dir= i.split()[0]
+            break
+        except:
+            pass
+        try:
+            bashCommand = "./icav2 projectdata list --parent-folder %sgrandeur/" % i.split()[0]
+            tmp= icav_out(bashCommand)
+            target_dir= i.split()[0]
+            break
+        except:
+            pass
+    mycosnp_files=['Results/stats/*','Results/multiqc_report.html','Results/combined/*']
+    Grandeur_files=['grandeur/*']
```

Comment:
Github's linter would like Grandeur_files to become grandeur_files (all lowercase). I don't think you need to change it, though.